adapters/Unity/UnityAdapter/GADMediationAdapterUnity.m

```diff
@@ -125,16 +138,30 @@ - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConf
   }
   self.adapterProxy = [[GADMUnityBannerMediationAdapterProxy alloc] initWithAd:self
                                                              completionHandler:completionHandler];
-
   self.bannerView = [[UADSBannerView alloc] initWithPlacementId:self.placementId
                                                            size:supportedSize.size];
   self.bannerView.delegate = self.adapterProxy;
-  [self.bannerView load];
+  UADSLoadOptions *loadOptions = [UADSLoadOptions new];
+  NSData *watermark = adConfiguration.watermark;
+  if (watermark != nil) {
+    NSString *watermarkString = [watermark base64EncodedStringWithOptions:0];
+    [loadOptions.dictionary setValue:watermarkString forKey:@"watermark"];
```

Comment:
nit: you might want to use the `GADMAdapterUnityWatermarkKey` here instead of a raw string